cmd/generate/generate.go

```diff
@@ -326,6 +331,43 @@ func main() {
 		if err := os.WriteFile(filepath.Join(output, "images.yaml"), mapping, fs.ModePerm); err != nil {
 			log.Fatal("Write images mapping file ", err)
 		}
+	} else if generators == GenerateMustGatherDockerfileOption {
```

Comment:
@pierDipi the reason for adding a new "generator" was, that the existing "dockerfile" generator is specialized on go projects. The must-gather seems to be only a Dockerfile parsing.
Or do you have another suggestion?